src/TraceEvent/Microsoft.Diagnostics.Tracing.TraceEvent.targets

```diff
@@ -0,0 +1,37 @@
+<!--
+***********************************************************************************************
+Microsoft.Bcl.Compression.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
```

Comment:
> Note that the previous Nuget package should be viewed as experimental. It should probably not be used as a template if there are better templates available. Ideally we start with a completely vanilla nuspec and tweek it just enough to get what we want.

I'm fine with that. I started with what existed to be less likely to "rock the boat". I'll work with @AArnott and/or @rainersigwald to recommend specific changes for a great long-term solution (dependable yet low maintenance).

> The thing that makes this package a bit harder than normal is that it has native components. 

My work was the foundation for libgit2sharp's native binaries distribution (libgit2/libgit2sharp#821), which has since been modified for improved cross-platform support ([libgit2/libgit2sharp.nativebinaries](https://github.com/libgit2/libgit2sharp.nativebinaries)). I plan to build on that work to the extent it relates to this project, but simplify it where it makes sense in cases that don't apply to us.

> Our goal is to NOT be innovative, but simply follow a standard for doing what we need (access native components).

There's a reason I suggested the two people I suggested for helping. :smile:

![image](https://cloud.githubusercontent.com/assets/1408396/26434582/1c030128-40cf-11e7-8705-490604d4da10.png) ![image](https://cloud.githubusercontent.com/assets/1408396/26434586/258b40c0-40cf-11e7-99bb-460778cb7a49.png)

> We may wish to split the package (symbol resolution, collection and processing are the obvious groupings).

This tends to cause more headaches than it solves. We should approach that if/when it becomes a clear requirement.